Fitfty/Projects/MainFeed/Sources/ViewControllers/BottomSheetViewController.swift

```diff
@@ -0,0 +1,208 @@
+        let view = UIView()
+        view.backgroundColor = .white
+        view.layer.cornerRadius = 20
+        view.layer.maskedCorners = [.layerMinXMinYCorner, .layerMaxXMinYCorner]
+        view.clipsToBounds = true
+        view.addSubviews(contentViewController.view)
+        return view
+    }()
+    
+    public init(
+        style: ModalStyle,
+        contentViewController: UIViewController,
+        coordinator: MainCoordinatorInterface
+    ) {
+        self.style = style
+        self.contentViewController = contentViewController
+        self.coordinator = coordinator
+        super.init(nibName: nil, bundle: nil)
+        
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    public override func viewDidLoad() {
+        super.viewDidLoad()
+        setUp()
+        
+    }
+    
+    public override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        showBottomSheet()
+    }
+}
+
+private extension BottomSheetViewController {
+    
+    func setUp() {
+        setUpContentView()
+        setUpLayout()
+        setUpGestureRecognizer()
+    }
+    
+    func setUpContentView() {
+        addChild(contentViewController)
+        contentViewController.didMove(toParent: self)
+    }
+    
+    func setUpLayout() {
+        view.backgroundColor = .clear
+        view.addSubviews(backgroundView, bottomSheetView)
+        bottomSheetViewTopConstraint = bottomSheetView.topAnchor.constraint(
+            equalTo: view.topAnchor,
+            constant: view.bounds.height
+        )
+        bottomSheetViewTopConstraint.priority = .defaultLow
+        NSLayoutConstraint.activate([
+            bottomSheetViewTopConstraint,
+            bottomSheetView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            bottomSheetView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            bottomSheetView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            backgroundView.topAnchor.constraint(equalTo: view.topAnchor),
+            backgroundView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            backgroundView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            backgroundView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            contentViewController.view.topAnchor.constraint(equalTo: bottomSheetView.topAnchor),
+            contentViewController.view.leadingAnchor.constraint(equalTo: bottomSheetView.leadingAnchor),
+            contentViewController.view.trailingAnchor.constraint(equalTo: bottomSheetView.trailingAnchor),
+            contentViewController.view.bottomAnchor.constraint(equalTo: bottomSheetView.bottomAnchor)
+        ])
+    }
+    
+    func setUpGestureRecognizer() {
+        let dimmedTap = UITapGestureRecognizer(target: self, action: #selector(didTapDimmedView(_:)))
+        view.addGestureRecognizer(dimmedTap)
```

Comment:
```suggestion
        backgroundView.addGestureRecognizer(dimmedTap)
```

[p3] 현재 bottomSheet를 눌러도 바로 dismiss되게 동작하는데 
이보단 뒤에 backgroundView 탭할 시에만 dismiss되는 동작이 더 맞는 것 같습니다😁

Reply:
헉.. 제가 실수했네용,,,!! 발견해주셔서 감사해용 🥹
https://github.com/YAPP-Github/21st-iOS-Team-2-iOS/pull/10/commits/e8d79c165cf32554293344a70ce445fa3b76f125